### DISCUSSION TITLE
datablocks - node based data wrangling

### DISCUSSION THREAD

**Discussion by moklick** [xyflow/xyflow#1027] [Show and tell]
![db](https://user-images.githubusercontent.com/2857535/112030127-38aa9700-8b3a-11eb-8306-2036db656d8b.jpg)

## datablocks - node based data wrangling

We are using React Flow to build a node based data wrangling editor called datablocks.

You can find all information on [datablocks.pro](https://datablocks.pro/).

**Comment by mikebarkmin** [comment]
Great looking UI. Are the headings + descriptions custom nodes with a transparent background or are they attached to the nodes and edges below?

**Reply by chrtze** [reply]
Hi! Thanks for the feedback. These are individual markdown nodes with transparent background, not attached to the other nodes.

**Comment by tony** [comment]
Hey when you're ready to 🚀  remember to post on https://news.ycombinator.com (Show HN) and reddit.  Looking good, keep up the momentum!

**Comment by gsaivinay** [comment]
Hello,

I was wondering what is the status of this project. Would this be open-sourced?

Thanks.

**Reply by moklick** [reply]
Currently we are working on other things. As soon as there is a plan you can read about it here https://reactflow.dev/blog/

**Reply by marcantoinethimm** [reply]
@moklick I noticed that you can't login / register , are there more features than the one showed in the demo once logged in ?

**Comment by ries9112** [comment]
Super cool stuff on datablocks 👏 hoping one day some small parts of this get open sourced 😁

**Comment by channely** [comment]
we need it
